src/components/Slider/Slider.mdx

```diff
@@ -8,7 +8,7 @@ import Link from "../Link/Link";
 # Slider
 
 | Component Version | DS Version |
-| ----------------- | ---------- |
```

Comment:
Another trivial, accidental formatting update to an irrelevant file. LMK if I should take it out.